elastic/kibana
# [Console] Disable the monaco migration for 8.15 release

## Summary

This PR disables the Monaco Console in 8.15 as we don't want to add it to public release yet (it's planned to be released in 8.16).

**How to test:**
1. Check out the PR and start Es and Kibana
2. Verify that Console still uses the old Ace editor.
